### PR TITLE
Update Equipment Extended Database CSV file

### PR DESCRIPTION
**Pull request by sixlettervariables**
This updates the equipment extended database to match the formatting changes requested by @HammerGS. This also fixes a bug in `TechAdvancement::getExtinctionDateName`.

This also adds F_CLAN to the extinction factions on IS Medium Laser as an example (correct) usage of the previously unused method.

### CODE REVIEW

**Review by HammerGS — APPROVED** [review]
The outputs are exactly what was needed.

File: megamek/src/megamek/common/TechAdvancement.java [hunk 1]
```diff
@@ -537,6 +537,20 @@ private String formatDate(int index, boolean clan, int[] factions) {
...
 
+    /**
+     * Formats introduction date indicating approximate when appropriate, and prototype faction if any
+     * for either IS or Clan use tech base.
+     */
+    public String getIntroductionDateName() {
+        if (getPrototypeDate() > 0) {
+            return getPrototypeDateName();
+        }
+        if (getProductionDate() > 0) {
```
Comment (Windchild292) [inline megamek/src/megamek/common/TechAdvancement.java:548]:
```suggestion
        } else if (getProductionDate() > 0) {
```

File: megamek/src/megamek/common/TechAdvancement.java [hunk 2]
```diff
@@ -537,6 +537,20 @@ private String formatDate(int index, boolean clan, int[] factions) {
...
+            return getProductionDateName();
+        }
+        return getCommonDateName();
```
Comment (Windchild292) [inline megamek/src/megamek/common/TechAdvancement.java:551]: Move into an else statement, like getExtinctionDateName

**Review by Windchild292 — APPROVED** [review]
Why the addition of quotation marks?

**Comment by sixlettervariables** [comment]
These are CSV files and many of these data points had commas that needed to be quoted.

**Comment by neoancient** [comment]
Some equipment has a different intro date for IS and Clan, and only the first is showing.

**Comment by sixlettervariables** [comment]
We'd discussed that and didn't have a great solution for the CSV format. Suggestions?

**Comment by neoancient** [comment]
You could try squeezing both into one field. But if Hammer's ok with it like this, I'd say it's fine.

**Review by neoancient — APPROVED** [review]

**Pull request merged**